<ui version="4.0" >
 <class>QListenRisRequestsConfigurationScreenBase</class>
 <widget class="QWidget" name="QListenRisRequestsConfigurationScreenBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RIS Listener Configuration</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_buttonApplyListenRisConfiguration" >
       <property name="text" >
        <string>Apply Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="m_checkBoxViewAutomaticallyStudies" >
       <property name="text" >
        <string>Automatically view the studies upon reception</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="m_checkBoxListenRisRequests" >
       <property name="text" >
        <string>Listen RIS Requests on port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_textPortListenRisRequests" >
       <property name="maximumSize" >
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_checkBoxListenRisRequests</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_textPortListenRisRequests</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>119</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel" >
     <x>229</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkBoxListenRisRequests</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_checkBoxViewAutomaticallyStudies</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>119</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel" >
     <x>229</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
